translations/handlePlainTranslations.sh

```diff
@@ -37,6 +37,13 @@ if [ -d src/main/res ]; then
   mv src/main/res/values-de-rDE src/main/res/values-de
 fi
 
+# move language folder as java uses old locales
+if [ -d src/main/res ]; then
+  mv src/main/res/values-he src/main/res/values-iw
+  mv src/main/res/values-id src/main/res/values-in
+  mv src/main/res/values-yi src/main/res/values-ji
```

Comment:
Just add them to the transifex config: https://github.com/nextcloud/android/blob/master/.tx/config#L9